### PR TITLE
Breakpoints restructuring

### LINKED ISSUE #2948: GEL Breakpoints' MEDIA_QUERY_TYPOGRAPHY object not reflective of device sizes
**Is your feature request related to a problem? Please describe.**
The current `LAPTOP_AND_LARGER` `MEDIA_QUERY_TYPOGRAPHY` breakpoint within `packages/utilities/gel-foundations/src/breakpoints.js` has the min-width `GEL_GROUP_CD_MIN_WIDTH`, which is 600px. The UX team use a 600px break point to describe a tablet or a very small notebook, not a laptop. There is no `MEDIA_QUERY_TYPOGRAPHY` breakpoint for 1008px, which is the 'laptop and larger' measurement used by designers.

**Describe the solution you'd like**

- Rename the `LAPTOP_AND_LARGER` `MEDIA_QUERY_TYPOGRAPHY` object key to `LARGE_SMARTPHONE...` or `TABLET...` `..._AND_LARGER`
- Add another key-value pair to the `MEDIA_QUERY_TYPOGRAPHY` for `LAPTOP_AND_LARGER` using a 1008px const.
- Add a final key-value pair for a 1280px breakpoint, which is the largest size the designers tend to work towards.

**Testing notes**
[Tester to complete]

**Dev insight:**
- The current `LAPTOP_AND_LARGER` 600px breakpoint is being used in 19 places, so this update will need to be matched with changes across the rest of the component library to reflect the new break point values.

- [ ] This feature is expected to need manual testing.

**Additional context**
Example of design breakpoints

image-breakpoints-examples

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #2948

**Overall change:** A small reorganisation of the `packages/utilities/gel-foundations/src/breakpoints.js` component, to better separate the GEL Grid breakpoints and the GEL Typography breakpoints. 

**Code changes:**

- Moved the block of consts used in the `MEDIA_QUERY_TYPOGRAPHY` object down below the grid consts
- Commented sizes of each REM for ease of use.
- Added reference to gel typography guidelines.

---

- [x] I have assigned myself to this PR and the corresponding issues

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]
👍

File: packages/utilities/gel-foundations/src/breakpoints.js [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-   These namings are based on the GEL description. They are also known as group A, group B and group D
-   Link to relevant GEL docs: http://www.bbc.co.uk/gel/guidelines/typography#type-sizes
-*/
-export const GEL_GROUP_A_MAX_WIDTH = 19.9375; // 319px
-export const GEL_GROUP_B_MIN_WIDTH = 20; // 320px
-export const GEL_GROUP_B_MAX_WIDTH = 37.4375; // 599px
-export const GEL_GROUP_CD_MIN_WIDTH = 37.5; // 600px
-
 /*
+    Breakpoints documentation: https://confluence.dev.bbc.co.uk/display/WSFrontPGS/Screen+sizes+and+Breakpoints
```
Comment (amywalkerdev) [inline packages/utilities/gel-foundations/src/breakpoints.js]: Probably not useful to link to our internal confluence as external contributors/users won't be able to view it. Does it contain any extra info that isn't inferred from below?

Comment (RichardPK) [reply packages/utilities/gel-foundations/src/breakpoints.js]: Good point. The confluence doc refers more explicitly to the const names that are in the file - ('Group A', 'Group B', 'Group C+D') so I thought it was more useful, but didn't consider its lack of accessibility for externals. Have removed!

**Review by DenisHdz — APPROVED** [review]
Apart from @amywalker1987's comment, looks good 👍

**Review by amywalkerdev — APPROVED** [review]
Thanks, looks good 👍

**Pull request merged**